lib/pedicel/validator.rb

```diff
@@ -120,49 +128,55 @@ def self.messages
 
       required('paymentDataType').filled(:str?, included_in?: %w[3DSecure EMV])
 
-      required('paymentData').schema do
-        optional('onlinePaymentCryptogram').filled(:str?, :base64?)
-        optional('eciIndicator').filled(:str?, :eci?)
-
-        optional('emvData').filled(:str?, :base64?)
-        optional('encryptedPINData').filled(:str?, :hex?)
-      end
+      required('paymentData').schema(TokenDataPaymentDataSchema)
 
       rule('paymentDataType affects paymentData': [:paymentDataType, [:paymentData, :onlinePaymentCryptogram]]) do |t, cryptogram|
         t.eql?('3DSecure') > cryptogram.filled?
       end
     end
 
-    def self.validate_token(token)
-      validation = TokenSchema.call(token)
+    class Error < StandardError; end
 
-      raise TokenFormatError, format_errors(validation) if validation.failure?
+    private
```

Comment:
The `private` statement here, and the following `public` statement, have no effect on visibility.
It's hard to find references to visibility of submodules in Ruby, but empirically scope doesn't change.